legal-api/tests/unit/reports/test_report_v2.py

```diff
@@ -21,14 +21,15 @@
 
 
 @pytest.mark.parametrize(
-    'test_name, variant', [
-        ('COMMENCEMENT_DEFAULT', 'default'),
-        ('COMMENCEMENT_GREYSCALE', 'greyscale'),
+    'test_name, variant, legal_type', [
+        ('COMMENCEMENT_DEFAULT_DISSOLUTION', 'default', 'BEN'),
+        ('COMMENCEMENT_GREYSCALE_DISSOLUTION', 'greyscale', 'BC'),
+        ('COMMENCEMENT_GREYSCALE_CANCELLATION', 'greyscale', 'A'),
     ]
```

Comment:
should add a test for default(color) EP scenario

Reply:
Added